src/components/ArticleView.jsx

```diff
@@ -0,0 +1,42 @@
+ import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { fetchArticleById } from '../../api/requests'
+ 
+ 
+ const ArticleView = () => {
+   
+    const {article_id} = useParams()
+    const [article,setArticle] = useState({})
```

Comment:
Please add a state and indicate loading for the user.

Reply:
Done. I have implemented that in my current working branch.